packages/persist/lib/records.ts

```diff
@@ -134,10 +134,10 @@ export async function persistRecords({
         }, 0);
 
         metrics.increment(metrics.Types.BILLED_RECORDS_COUNT, new Set(summary.billedKeys).size, { accountId });
-        metrics.increment(metrics.Types.PERSIST_RECORDS_COUNT, records.length, { accountId });
-        metrics.increment(metrics.Types.PERSIST_RECORDS_SIZE_IN_BYTES, recordsSizeInBytes, { accountId });
-        metrics.increment(metrics.Types.PERSIST_RECORDS_MODIFIED_COUNT, allModifiedKeys.size, { accountId });
-        metrics.increment(metrics.Types.PERSIST_RECORDS_MODIFIED_SIZE_IN_BYTES, modifiedRecordsSizeInBytes, { accountId });
+        metrics.increment(metrics.Types.PERSIST_RECORDS_COUNT, records.length);
+        metrics.increment(metrics.Types.PERSIST_RECORDS_SIZE_IN_BYTES, recordsSizeInBytes);
```

Comment:
Not sure if we want to remove the account dimension for this one (purposely excluded it from the list) because we might want to detect customers saving heavy records, no?

Reply:
ok. keeping the dimension for `PERSIST_RECORDS_SIZE_IN_BYTES` metrics